eip-4626.md

```diff
@@ -0,0 +1,155 @@
+# Yield Bearing Vault Standard
+
```

Comment:
Please follow the template found at https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md

The formatting on this header is incorrect.

Reply:
done